### DISCUSSION TITLE
トラックボール基盤を接続しているときに一部のキーが反応しない

### DISCUSSION THREAD

**Discussion by Soule50431** [Yowkees/keyball#255] [Q&A]
キーボール61を購入し作成いたしました。

ほとんどのキーは問題なく動作しているのですが、
トラックボールがある方のロープロファイル仕様の親指のキー２つが反応しません。
色々試した結果、トラックボール基盤を接続しているときにのみキーが反応しなくなるということがわかりました。
トラックボール基盤を外せば、問題なくキーは反応しております。

自作キーボードは初めてでどうすればよいかわからないので、
解決方法を提示していただけますと幸いです。

![unnamed](https://github.com/Yowkees/keyball/assets/57452379/3bcf6ad4-53f5-42b6-bce9-b6456c1da4e7)
![unnamed](https://github.com/Yowkees/keyball/assets/57452379/30743dfe-ef46-4cad-99db-bf36892f6345)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
症状から推測すると、トラックボールの信号線とキーの信号線がショートしていそうです。

下記画像の赤枠で示した端子の中で隣とショートしているものがいないかご確認ください。
ProMicro裏の半田付けは端子がジグザグに配置されていて、端子同士が近いのでショートしやすいです。

お手数ですがご確認お願いいたします。
![KB61R_KEYfix](https://github.com/Yowkees/keyball/assets/84209772/15461270-68b4-48ac-b493-54d86d3c77eb)
